packages/paste-design-tokens/tokens/themes/dark/global/text-color.yml

```diff
@@ -23,11 +23,14 @@ props:
     value: "{!palette-gray-0}"
     comment: Inverse text color for dark backgrounds. Must pass AA color contrast with color-background-body-inverse.
   color-text-inverse-weak:
-    value: "{!palette-gray-40}"
+    value: "{!palette-gray-30}"
     comment: Weak inverse text color for dark backgrounds. Must pass AA color contrast with color-background-body-inverse.
   color-text-inverse-weaker:
-    value: "{!palette-gray-70}"
+    value: "{!palette-gray-50}"
     comment: Weaker inverse text color for dark backgrounds. Must pass AA color contrast with color-background-body-inverse.
+  color-text-inverse-weakest:
+    value: "{!palette-gray-70}"
+    comment: Weakest inverse text color for dark backgrounds. Must pass AA color contrast with color-background-body-inverse.
```

Comment:
suggestion: can delete these lines since we are inheriting from the base text-color.yml and there aren't any changes afaict.

Reply:
can do 👍  they were added by robb so I kept them lol